tool/site/server_hapi/src/routes/dispense/withdraw.ts

```diff
@@ -0,0 +1,29 @@
+import Hapi from "@hapi/hapi"
+import {getSessionValue, removeFromSessionValue} from "../../services/session"
+import {Task} from "fhir/r4"
+import {getEpsClient} from "../../services/communication/eps-client"
+
+export default [
+  {
+    method: "POST",
+    path: "/dispense/withdraw",
+    handler: async (request: Hapi.Request, responseToolkit: Hapi.ResponseToolkit): Promise<Hapi.ResponseObject> => {
+      const releaseRequest = request.payload as Task
```

Comment:
i think this variable name is a little confusing. the payload that is sent to this endpoint is what we send to eps, which is a withdraw message. where does release come into it?